src/components/Header.vue

```diff
@@ -103,7 +103,7 @@ export default {
     //console.log(types_by_host);
 
     _.each(types_by_host, (types, hostname) => {
-      if (types.afk && types.window) {
+      if (hostname != 'unknown') {
```

Comment:
Looks like this makes the `_.each` over types unnecessary?

Reply:
Not sure due to the android related code, don't want to break it by mistake.

Reply:
The Android view is getting merged with the normal view in https://github.com/ActivityWatch/aw-webui/pull/183 anyway, so feel free to remove that.

Reply:
I think it'll be easier if you fix it in that change instead.